Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py

```diff
@@ -424,17 +424,6 @@ def test_elasticsearch_builder_called_with_username_password(mocker):
     assert es_mock.call_args[1].get('api_key') is None
 
 
-@patch("Elasticsearch_v2.API_KEY_ID", "demisto")
```

Comment:
Removed due to a change in the way the file is loaded - this causes issues testing this specific behavior, and as such the test isn't helpful.